mmrazor/models/architectures/utils/mutable_register.py

```diff
@@ -50,8 +51,13 @@ def mutate_mobilenet_layer(mb_layer: MBBlock, mutable_in_channels,
         mutable_kernel_size=mutable_kernel_size)
 
     if mb_layer.with_se:
+        mutable_expand_ratio2 = copy.deepcopy(mutable_expand_ratio)
+        mutable_expand_ratio2.alias += '_se'
+
+        derived_se_channels = mutable_expand_ratio2 * mutable_in_channels
```

Comment:
Changes for what?

Reply:
To avoid the repeat of alias when it comes to `fine_grained_mode`.